### PR TITLE
Actions reducers and Store

### PR DESCRIPTION
**Pull request by luccamicale**
## BookStore; add the actions, reducers, and store 💯 

##https://github.com/luccamicale/bookstore

## Checklist before requesting a review 🔢 🥇 
- [x] I have performed a self-review of my code
- [x] In this project I created actions inside the books and categories
- [x] Then I created the reducers to can change the current state
- [x] Finally, I created a store when there are combined actions and reducers. 👍

### CODE REVIEW

File: src/redux/books/boooks-redux.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+// action to delete
+const removeBook = () => ({
+  type: REMOVE_BOOK,
+  payload: test,
+});
+
+// create a reducer
+const initialState = (state = '', action) => {
+  if (action.type === ADD_BOOK) {
+    return action.payload;
```
Comment (topeogunleye) [inline src/redux/books/boooks-redux.js]: - [x] As required, in the case of where `action.type` is `ADD_BOOK` the function should return a new state object in which the books array will contain a new book object, passed by action.payload. You're not returning a new state below. See a Correct Example below:

```
  case ADD_BOOK:
      return [
        ...state,
        action.payload,
      ];

```

File: src/redux/books/boooks-redux.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  }
+  if (action.type === REMOVE_BOOK) {
+    return action.payload;
```
Comment (topeogunleye) [inline src/redux/books/boooks-redux.js]: - [x] As required, in the case of where `action.type` is `REMOVE_BOOK`, the function should ES6 filter() method to create a new array, which will not contain the book you want to remove from the store

## EXAMPLE
```
    case REMOVE_BOOK:
      return state.filter((item) => item.item_id !== action.payload.item_id);
```

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @luccamicale, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Try-Again

> ## (Highlights) Good Points: 💪 
> * Good commit messages. :heavy_check_mark:
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
> * Professional README file. :heavy_check_mark:
> * PR has a good title and summary :heavy_check_mark:

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Nothing to mention.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by luccamicale** [comment]
I modified the two problems in my project that he ask me for the last reviewer. Now my project should be approved!!! 💯 
🥇 🔢 👍

**Review by topeogunleye — APPROVED** [review]
### Hi @luccamicale, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and description :heavy_check_mark:
 
## [Optional] suggestions:
* Nothing to mention.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**